src/views/templates/details/tabs/scheduling/components/Descheduler.tsx

```diff
@@ -1,25 +1,30 @@
 import React from 'react';
-import { TemplateSchedulingGridProps } from 'src/views/templates/details/tabs/scheduling/components/TemplateSchedulingLeftGrid';
 import { useDeschedulerInstalled, useDeschedulerOn } from 'src/views/templates/utils';
 import { DESCHEDULER_URL } from 'src/views/templates/utils/constants';
 
 import { useModal } from '@kubevirt-utils/components/ModalProvider/ModalProvider';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { V1Template } from '@kubevirt-utils/models';
 import {
   Button,
   DescriptionListDescription,
   DescriptionListGroup,
   DescriptionListTermHelpText,
   DescriptionListTermHelpTextButton,
   Popover,
+  Tooltip,
 } from '@patternfly/react-core';
 import { ExternalLinkAltIcon, PencilAltIcon } from '@patternfly/react-icons';
 
 import DeschedulerModal from './DeschedulerModal';
 
 import 'src/views/templates/details/tabs/scheduling/TemplateSchedulingTab.scss';
 
-const Descheduler: React.FC<TemplateSchedulingGridProps> = ({ template }) => {
+type DeschedulerProps = {
+  template: V1Template;
+};
+
+const Descheduler: React.FC<DeschedulerProps> = ({ template }) => {
   const { t } = useKubevirtTranslation();
   const { createModal } = useModal();
   const isDeschedulerOn = useDeschedulerOn(template);
```

Comment:
this hook is a bug, when checking for descheduler on/off you should only check for annotation, not looking for operators.

Reply:
We need to check for both, the operator and the annotation.